webui/react/src/components/FilterForm/components/FilterField.tsx

```diff
@@ -157,6 +157,16 @@ const FilterField = ({
     },
   });
 
+  const captureEnterKeyDown = useCallback(
+    (e: React.KeyboardEvent) => {
+      // would use isComposing alone but safari has a bug: https://bugs.webkit.org/show_bug.cgi?id=165004
+      if (e.key === 'Enter' && (!e.nativeEvent.isComposing || e.keyCode === 229)) {
```

Comment:
`keyCode` should be `13`

The current behavior

https://github.com/determined-ai/determined/assets/109113616/5bbcafa4-dcd6-4267-a882-7631910f2ee2

Reply:
sorry, i think the core issue is that we want to ignore the press during composition, but I forgot to distribute the negative check. Pushed a fix here.